src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -23,7 +23,7 @@
 -->
 
 <template>
-	<div v-bind="filePreview"
+	<element v-bind="filePreview"
```

Comment:
Shouldn't we use `<component />` here?

https://vuejs.org/api/built-in-special-elements#component

Reply:
Sure, this is not really an official standard or recommendation. `component` or `element` is fine for me :shrug:

Reply:
👍 Then I'll change a tag name and squash, if you don't mind

Reply:
So time to get rid of the `fragment` library we have?

Reply:
> to get rid

No, it has different usage (although I'm not the fan of it). `<Fragment>` allows to put several elements as a component root:

```vue
<template>
  <Fragment>
    <div class="1"/>
    <div class="2"/> 
  <Fragment>
<template>
```

`<component>` can render different components upon conditions:
```vue
  <component :is="isSystemMessagesGroup ? MessagesSystemGroup : MessagesGroup" />
```